system/Database/BaseBuilder.php

```diff
@@ -86,8 +86,10 @@ class BaseBuilder
 
     /**
      * QB keys
+     * list of column names.
      *
      * @var array
+     * @phpstan-var list<string>
```

Comment:
Just for my own edification... what is the benefit of using the vendor-specific annotation? I know Psalm and some IDEs understand `@var string[]` - won't they miss out on this nuance?

Reply:
`string[]` means `array<string>`. It does not care about keys.
https://psalm.dev/docs/annotating_code/type_syntax/array_types/#phpdoc-syntax

`list` means true list that the key starts with 0, and goes 1,2,3,...

Reply:
VS Code does not support it yet.